app/code/Magento/Catalog/Model/Product/Link/SaveHandler.php

```diff
@@ -18,7 +18,7 @@ class SaveHandler
     /**
      * @var ProductLinkRepositoryInterface
      */
-    protected $productLinkRepository;
+    protected $productLinkRepo;
```

Comment:
Please don't rename a protected property as this breaks backwards compatibility ([source](http://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/index.html)). I know you get a warning regarding the variable name length, but backwards compatibility is more important than code style.